scipy/stats/_continuous_distns.py

```diff
@@ -2039,7 +2039,10 @@ def _ppf(self, q, a, c):
         return where(cond > 0, val1**ic, val2**ic)
 
     def _munp(self, n, a, c):
-        return special.gamma(a+n*1.0/c) / special.gamma(a)
+        # return special.gamma(a+n*1.0/c) / special.gamma(a)
```

Comment:
Remove this line; there is no need to keep code that has been commented-out.  The comment below is a sufficient reminder of the meaning of `poch`.